src/installer/test/Microsoft.Extensions.DependencyModel.Tests/Microsoft.Extensions.DependencyModel.Tests.csproj

```diff
@@ -17,8 +17,8 @@
     <PackageReference Include="Moq" Version="4.7.142" />
     <PackageReference Include="NuGet.Versioning" Version="4.0.0" />
     <PackageReference Include="Microsoft.DotNet.ProjectModel" Version="1.0.0-rc2-002702" />
-    <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonVersion)" />
-    <PackageReference Include="System.Text.Encodings.Web" Version="$(SystemTextEncodingsWebVersion)" />
+    <PackageReference Include="System.Text.Json" Version="4.7.0" />
```

Comment:
I assume this reference can be deleted because it should come transitively from the `ProjectReference` to the DependencyModel library. But you don't have to worry about it now. I'll open a PR to remove it separately.